src/scales/scale.radialLinear.js

```diff
@@ -404,8 +405,8 @@ export default class RadialLinearScale extends LinearScaleBase {
     }
   }
 
-  getPointPosition(index, distanceFromCenter) {
-    const angle = this.getIndexAngle(index) - HALF_PI;
+  getPointPosition(index, distanceFromCenter, {additionalAngle = 0} = {}) {
```

Comment:
Since you only expect a singel value as extra argument it would be better for performance to just pass that value instead of a object that has to be destructured.

Reply:
I changed the argument to a single value.